sdk/keyvault/keyvault-admin/src/backupClient.ts

```diff
@@ -0,0 +1,277 @@
+   * @param credential An object that implements the `TokenCredential` interface used to authenticate requests to the service. Use the @azure/identity package to create a credential that suits your needs.
+   * @param [pipelineOptions] Pipeline options used to configure Key Vault API requests. Omit this parameter to use the default pipeline configuration.
+   */
+  constructor(
+    vaultUrl: string,
+    credential: TokenCredential,
+    pipelineOptions: BackupClientOptions = {}
+  ) {
+    this.vaultUrl = vaultUrl;
+
+    const libInfo = `azsdk-js-keyvault-admin/${SDK_VERSION}`;
+
+    const userAgentOptions = pipelineOptions.userAgentOptions;
+
+    pipelineOptions.userAgentOptions = {
+      ...pipelineOptions.userAgentOptions,
+      userAgentPrefix:
+        userAgentOptions && userAgentOptions.userAgentPrefix
+          ? `${userAgentOptions.userAgentPrefix} ${libInfo}`
+          : libInfo
+    };
+
+    const authPolicy = isTokenCredential(credential)
+      ? challengeBasedAuthenticationPolicy(credential)
+      : signingPolicy(credential);
+
+    const internalPipelineOptions = {
+      ...pipelineOptions,
+      ...{
+        loggingOptions: {
+          logger: logger.info,
+          logPolicyOptions: {
+            allowedHeaderNames: [
+              "x-ms-keyvault-region",
+              "x-ms-keyvault-network-info",
+              "x-ms-keyvault-service-version"
+            ]
+          }
+        }
+      }
+    };
+
+    const pipeline = createPipelineFromOptions(internalPipelineOptions, authPolicy);
+    this.client = new KeyVaultClient({
+      apiVersion: pipelineOptions.serviceVersion || LATEST_API_VERSION,
+      ...pipeline
+    });
+  }
+
+  /**
+   * Starts generating a backup of an Azure Key Vault on the specified Storage Blob account.
+   *
+   * This function returns a Long Running Operation poller that allows you to wait indefinitely until the Key Vault backup is generated.
+   *
+   * Example usage:
+   * ```ts
+   * const client = new KeyVaultBackupClient(url, credentials);
+   *
+   * const blobStorageUri = "<blob-storage-uri>"; // <Blob storage URL>/<folder name>
+   * const sasToken = "<sas-token>";
+   * const poller = await client.beginBackup(blobStorageUri, sasToken);
+   *
+   * // Serializing the poller
+   * const serialized = poller.toString();
+   * // A new poller can be created with:
+   * // await client.beginBackup(blobStorageUri, sasToken, { resumeFrom: serialized });
+   *
+   * // Waiting until it's done
+   * const backupUri = await poller.pollUntilDone();
+   * console.log(backupUri);
+   * ```
+   * @summary Creates a new role assignment.
+   * @param blobStorageUri The URL of the blob storage resource, with the path to the folder name where the backup will end up being generated.
+   * @param sasToken The SAS token.
+   * @param [options] The optional parameters.
+   */
+  public async beginBackup(
```

Comment:
python has it as `begin_full_backup`

Reply:
.Net has it as startBackup https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs#L50

Reply:
cc: @christothes , @heaths

Reply:
In JS we favor "begin". Assuming that this prefix is fine, what I'm most concerned is of the `full` part.

Reply:
I don't have a strong opinion here, but I opted for plain `backup` because there is no other backup operation to contrast with.